docs/docs/03-syntax-and-usage/11-script-templates.md

```diff
@@ -103,3 +103,50 @@ func main() {
 	}
 }
 ```
+
+You can also directly render the Javascript function, passing in Go data, using the `!` expression:
+
+```templ
+package main
+
+script printToConsole(content string) {
+	console.log(conent)
+}
+
+templ page(content string) {
+	<html>
+		<body>
+		  {! printToConsole(content) }
+		</body>
+	</html>
+}
+```
+
+The data passed into the Javascript funtion will be JSON encoded, which then can be used inside the function.
+
+```go
```

Comment:
This is great, in that it shows how to use it, but it doesn't show the expected HTML output, which I think is really useful.

A lot of people aren't going to copy and paste the example and try it out, so giving the example output makes it a lot less abstract when people are skimming through.

Reply:
Added the output